Plots/WRF/NCL_dataonmap_10.py

```diff
@@ -0,0 +1,99 @@
+"""
+NCL_dataonmap_10_lg.py
+======================
+This script illustrates the following concepts:
+    - Plotting WRF data on native grid
+    - Plotting data using wrf python functions
+    - Overlaying continent outlines on a map
+    - Following best practices when choosing a colormap.
+      More information on colormap best practices can be found `here <https://geocat-examples.readthedocs.io/en/latest/gallery/Colors/CB_Temperature.html#sphx-glr-gallery-colors-cb-temperature-py>`_.
+    
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/dataonmap_10.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/dataonmap_10_1_lg.png
+"""
+
+###############################################################################
+# Import packages
+
+from netCDF4 import Dataset
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mticker
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+from wrf import (getvar, to_np, latlon_coords, get_cartopy)
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in the data
+
+wrfin = Dataset(gdf.get("netcdf_files/wrfout_d01_2003-07-15_00:00:00"), decode_times=True)
```

Comment:
Once the filename is changed in geocat-datafiles, we should update this example to use the name without colons so users have an OS agnostic example.

Reply:
I updated this in the most recent push, but you may have to do a `git pull` on the geocat-datafiles before it will reflect locally